container/containerd/client.go

```diff
@@ -125,6 +130,21 @@ func (c *client) Version(ctx context.Context) (string, error) {
 	return response.Version, nil
 }
 
+func (c *client) ContainerFsUsage(ctx context.Context, snapshotter, snapshotkey string) (*common.FsUsage, error) {
```

Comment:
does `ContainerFsUsage` need to be exported?

Reply:
yes , this client  is delivered as interface, if this method is not exported, we can't get the fs usage.

```go
func newContainerdContainerHandler(
	client ContainerdClient,//  Here
	name string,
	machineInfoFactory info.MachineInfoFactory,
	fsInfo fs.FsInfo,
	cgroupSubsystems *containerlibcontainer.CgroupSubsystems,
	inHostNamespace bool,
	metadataEnvs []string,
	includedMetrics container.MetricSet,
) (container.ContainerHandler, error)
```